sdk/src/main/java/com/bugsnag/android/Client.java

```diff
@@ -404,6 +402,7 @@ public void setEndpoint(String endpoint) {
      *
      * @param buildUUID the buildUUID.
      */
+    @SuppressWarnings("checkstyle:AbbreviationAsWordInName")
```

Comment:
The parameter to the function can change, even if the function name has to stay the same

Reply:
Thanks @Pezzah - I've updated the param name